src/alarms/notifications/EditNotification.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
+import MDSpinner from "react-md-spinner";
+
+import { NotificationForm } from "./NotificationForm";
+
+class EditNotificationModel extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            currentNotification: null
+        };
+    }
+
+    componentDidMount() {
+        const { match, alarmType } = this.props;
+
+        (async () => {
+            const url = (alarmType === "RASTERS") ? (
+                `/api/v4/rasteralarms/${match.params.id}/`
+            ) : (
+                `api/v4/timeseriesalarms/${match.params.id}/`
```

Comment:
One has a starting /, the other doesn't.

Reply:
I think this code should be in constructor, no need to have it in a separate componentDidMount function.

Reply:
Moved to inside constructor and added /